### ISSUE TITLE
Is it possible to change icon for the fab button?

### ISSUE THREAD

**Issue by slormin** [Nightonke/BoomMenu#82]
Something like add user icon, or a bell in the main boom menu button (the one that explodes :) )

**Comment by Nightonke** [comment]
Emm...
Add a custom icon on BMB itself is not supported. Because u know, sub-buttons boom from pieces to complete buttons. The pieces look strange on BMB with a custom icon.

**Comment by dengue8830** [comment]
i have a hack for this, only define a image view (this will be the icon for the user) and a boom menu button (with width and height at 0dp, visibility="gone" throws an error). Later in java add a listener to your clickeable element (the image view) and "bmb.boom();"

**Comment by Nightonke** [comment]
@dengue8830 Nice trick. I will add this feature in the next version of BMB.

**Comment by dengue8830** [comment]
Thanks bro

**Comment by kingmaker-agm** [comment]
Please add this feature quickly. I need to add this feature in my Android app.

Thanks & Regards,
[Kingmaker](http://kingmaker.co.in)

**Comment by Nightonke** [comment]
@slormin @dengue8830 @kingmaker-agm 
This feature is supported since version 2.1.0. Check [fade-view-wiki](https://github.com/Nightonke/BoomMenu/wiki/Fade-Views) for more details.

**Comment by dengue8830** [comment]
Great! Thanks you! Nice trick by de way ;)

**Comment by EricSzla** [comment]
Awesome feature.
Just a quick question, how could I remove the standard stripes image on the fab button ?
I am using the fade-view with my own image, but it overlays the stripes instead of replacing them. 

Thanks !

**Comment by kingmaker-agm** [comment]
@EricSzla, you have to use the xml attribute app:bmb_dotRadius="0" to hide the dots within the BoomMenuButton.

**Comment by EricSzla** [comment]
@kingmaker-agm thanks for the reply, this is how the button looks like in my xml, and yet the dots still appear on the button:

<com.nightonke.boommenu.BoomMenuButton
        android:id="@+id/bmb"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        **app:bmb_dotRadius="0dp"**
        app:bmb_buttonEnum="ham"
        app:bmb_normalColor="@ color/primary_color"
        android:layout_alignParentBottom="true"
        android:layout_alignParentEnd="true"

**Comment by kingmaker-agm** [comment]
@EricSzla, make sure that you were using the latest version of BoomMenu......

app:bmb_dotRadius is supported since 2.1.0

Check for the version in your build.gradle file

**Comment by EricSzla** [comment]
@kingmaker-agm I'm using 2.1.1, I've changed it to 2.1.0 and unfortunately still same result.

Thanks!

**Comment by DanielStankovic** [comment]
Did you manage to solve this? My guess is that **app:bmb_dotRadius="0dp"** works only if we use Circle button. On Ham buttons, the image overlaps the stripes.

**Comment by kingmaker-agm** [comment]
Yes, I am using Circular Buttons and **app:bmb_dotRadius="0dp"** works for me. I am not sure in other cases.

**Comment by mobile-suit-taurran** [comment]
For Ham buttons, use **app:bmb_hamHeight="0dp"** and **app:bmb_hamWidth="0dp"** instead of **app:bmb_dotRadius="0dp"**

**Comment by kushanshah11** [comment]
@slormin @dengue8830 @kingmaker-agm @Nightonke 
is there any way to set image when boom menu is open?

i have tried in this ways but no luck......

 <com.nightonke.boommenu.BoomMenuButton
        android:id="@+id/bmb"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="center"
        app:bmb_inFragment="true"
        app:bmb_dotRadius="0dp"
        app:bmb_normalColor="@color/primary_blue"
        app:bmb_rippleEffect="true"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintLeft_toRightOf="@+id/frameBook"
        app:layout_constraintStart_toStartOf="parent">

        <ImageView
            android:id="@+id/ivBoom"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:src="@drawable/btn_add"
            android:visibility="gone"/>
        
        <ImageView
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:src="@drawable/icon_close"
            android:layout_gravity="center"
            android:visibility="gone"/>

    </com.nightonke.boommenu.BoomMenuButton>


and on **setOnBoomListener** i have changed visibility for both buttons 

but cross icon visible for sometimes and  gone.

attach image for reference.

![image](https://user-images.githubusercontent.com/18567763/44090126-bcc2bcee-9fe6-11e8-82c2-3c362487928e.png)

please help!!!!

Thanks
